package.json

```diff
@@ -24,7 +24,7 @@
     "build": "node-gyp build",
     "readme": "jsdoc2md --template docs/README.hbs index.js > README.md",
     "lint-js": "eslint *.js",
-    "lint-cpp": "cpplint src/*",
+    "lint-cpp": "cpplint --recursive src/*",
```

Comment:
Really? Afaik singe asterisk wildcards work (i.e. `lib/*.js`), but maybe that was just a dream. Mended again :)